src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -337,34 +350,8 @@ public void afterNavigateTo(String url, WebDriver driver) {
                 .logInfo("Hack for disabling notifications", "Failed to execute js action");
 
       }
-      /**
-       * All of tests should be executed as an user who opted in (agreed) on using ads tracking.
-       * Manually user would need to click 'agree' in the tracking opt in modal.
-       */
 
-      if (TestContext.isFirstLoad()) {
-        boolean userOptedIn = true;
-        boolean userOptedOut = false;
-
-        if (method.isAnnotationPresent(Execute.class) && !method.getAnnotation(Execute.class)
-            .trackingOptIn()) {
-          userOptedIn = false;
-        }
-
-        if (method.isAnnotationPresent(Execute.class)) {
-          userOptedOut = method.getAnnotation(Execute.class).trackingOptOut();
-        }
-
-        if (userOptedIn) {
-          driver.manage().addCookie(
-              new Cookie("tracking-opt-in-status", "accepted", ".wikia.com", "/",
-                         new Date(new DateTime().plusYears(10).getMillis())));
-        } else if (userOptedOut) {
-          driver.manage().addCookie(
-              new Cookie("tracking-opt-in-status", "rejectedgit", ".wikia.com", "/",
-                         new Date(new DateTime().plusYears(10).getMillis())));
-        }
-      }
+//      handleGdprModalWithCookie();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)